src/client/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -311,6 +313,25 @@ public T WithExperimentalFeatures(bool enableExperimentalFeatures = true)
             return (T)this;
         }
 
+        /// <summary>
+        /// Microsoft Identity specific OIDC extension that allows resource challenges to be resolved without interaction. 
```

Comment:
> Microsoft Identity specific OIDC extension that allows resource challenges to be resolved without interaction.
I don't understand what this means.

Do we want to say something like:
enables client applications to advertise to the Microsoft identity platform endpoint that they are capable of handling some specific fields and behaviors, and therefore the response to acquire token can be more elaborated?